app/src/main/java/io/intrepid/contest/screens/sendinvitations/SendInvitationsActivity.java

```diff
@@ -142,8 +128,54 @@ public void onSelectContactsButtonClicked() {
         presenter.onSelectContactsButtonClicked();
     }
 
+    private void replaceFragment(Fragment fragment) {
+        getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.fragment_container, fragment)
+                .commitAllowingStateLoss();
```

Comment:
Because I was having trouble coming back to the activity from another app, after resetting the permissions; this way I don't use `savedInstanceState`